socorro/unittest/external/es/test_supersearch.py

```diff
@@ -109,28 +109,6 @@ def test_get(self):
         # processed_crash.json_dump.write_combine_size > write_combine_size
         assert 'write_combine_size' in res['hits'][0]
 
-    @minimum_es_version('1.0')
-    def test_get_with_root_field(self):
-        """Verify that querying fields at the root of the crash document works.
-        """
-        self.index_crash({
-            'signature': 'js::break_your_browser',
-            'date_processed': self.now,
-        }, root_doc={
-            'removed_fields': 'foo bar',
-        })
-        self.refresh_index()
-
-        res = self.api.get(_columns=[
-            'date', 'signature', 'removed_fields'
-        ], _facets=['removed_fields'])
-
-        assert 'removed_fields' in res['hits'][0]
-        assert res['hits'][0]['removed_fields'] == 'foo bar'
-
-        assert 'removed_fields' in res['facets']
-        assert len(res['facets']['removed_fields']) == 2
-
```

Comment:
We don't have anything in the "root document", so I nixed the test for this feature which we don't use.